imkar/scattering/analytical.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import pyfar as pf
+
+import numbers
+
+
+def rectangular(frequencies, incident_angles, rectangle_width, gap_width,
+                height, speed_of_sound=346.18):
+    """
+    This function computes the scattering coefficient of periodic rectangular
+    profiles, assuming that the extent is infinite.
```

Comment:
More in line with pyfar docstring conventions, if starting with a verb.
```suggestion
    Compute analytical scattering coefficient for infinite periodic rectangular profile.
```